src/otx/core/data/entity/visual_prompting.py

```diff
@@ -6,19 +6,22 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Type
 
+from datumaro import Polygon as dmPolygon
 from torchvision import tv_tensors
+from torchvision.tv_tensors import BoundingBoxes as tvBoundingBoxes
+from torchvision.tv_tensors import Mask as tvMask
 
 from otx.core.data.entity.base import OTXBatchDataEntity, OTXBatchPredEntity, OTXDataEntity, OTXPredEntity, Points
 from otx.core.data.entity.utils import register_pytree_node
 from otx.core.types.task import OTXTaskType
 
 if TYPE_CHECKING:
-    from datumaro import Polygon as dmPolygon
     from torch import LongTensor
-    from torchvision.tv_tensors import BoundingBoxes as tvBoundingBoxes
-    from torchvision.tv_tensors import Mask as tvMask
+
+
+ZeroShotPromptType = Type[tvBoundingBoxes | Points | tvMask | dmPolygon]
```

Comment:
```suggestion
ZeroShotPromptType = tvBoundingBoxes | Points | tvMask | dmPolygon
```
Isn't it unnecessary to use `Type` in this case?

Reply:
Mypy was upset without this.
error: Variable "otx.core.data.entity.visual_prompting.ZeroShotPromptType" is not valid as a type  [valid-type]
error: ZeroShotPromptType? has no attribute "pin_memory"  [attr-defined]
error: ZeroShotPromptType? has no attribute "cpu"  [attr-defined]